docs/example/basic.rst

```diff
@@ -359,14 +359,14 @@ use :ref:`generative-envlist` and :ref:`conditional settings <factors>` to expre
 .. code-block:: ini
 
     [tox]
-    envlist = py{27,34,36}-django{15,16}-{sqlite,mysql}
+    envlist = py{27,36,37}-django{22,30}-{sqlite,mysql}
```

Comment:
Can you switch to 39 and 38? 😄

Reply:
There is no official support of Django for py39

Reply:
Then let's go with 38, and 37. Instead of 36.